src/config.c

```diff
@@ -673,6 +752,18 @@ void loadServerConfigFromString(char *config) {
             server.client_obuf_limits[class].hard_limit_bytes = hard;
             server.client_obuf_limits[class].soft_limit_bytes = soft;
             server.client_obuf_limits[class].soft_limit_seconds = soft_seconds;
+        } else if (!strcasecmp(argv[0], "bgsave-cpu-mask") && argc == 2) {
+          cpu_set_t set;
+          if (strlen(argv[1]) == 0) {
+            zfree(server.bgsave_cpu_set);
+            server.bgsave_cpu_set = NULL;
+          } else if (parseCPUSet(argv[1], &set) != 0) {
+            err = "bad CPU mask"; goto loaderr;
```

Comment:
I guess the capitalization isn't consistent throughout, but it's a little weird that we have both `"bad CPU mask"` and `"Bad CPU mask"`.

Reply:
I thought I was copying the capitalization around me, but taking another look the lowercase one I saw was an outlier. I've changed it to uppercase as well.